main.py

```diff
@@ -140,8 +201,32 @@
                 # 3 - abducted      (no view, no type)
                 # 4 - dead          (dead role?)
 
-            # TODO
-            pass
+            # 0 -> read = False
+            # 1 -> read = True
+            # 2 -> give frozen (if they don't have it yet)
+            # 3 -> read = False
+            # 4 -> give dead role + remove participant role
+
+            channel = client.get_channel(element.channel)
+            user = client.get_user(element.victim)
+            if element.number == 0:
+                await channel.set_permissions(user, read_messages=False, send_messages=False)
+            elif element.number == 1:
+                await channel.set_permissions(user, read_messages=True, send_messages=True)
+            elif element.number == 2:
+                await channel.set_permissions(user, read_messages=True, send_messages=False)
+            elif element.number == 3:
+                await channel.set_permissions(user, read_messages=False, send_messages=False)
+            elif element.number == 4:
+                await channel.set_permissions(user, read_messages=True, send_messages=False)
```

Comment:
I thought we weren't explicitly setting read_messages since it caused problems last time

Reply:
Yes, but that was because send_messages _wasn't_ set. If this overrides one's usual permissions of the channel, this should be completely fine.